homeassistant/components/esphome/light.py

```diff
@@ -24,6 +25,9 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util.color import (
+    color_temperature_mired_to_kelvin as mired_to_kelvin,
+)
```

Comment:
```suggestion
```

Reply:
Import removed in commit 879575d1385c06588fbe2d86f39f22a71506e791